doc/sphinx/dev_git_intro.rst

```diff
@@ -52,46 +52,211 @@ Clone a local repository onto your machine
 .. (TODO: `pip install -v .`, other installation instructions...)
 
 Start coding
-^^^^^^^^^^^^
+------------
 
-- Switch to the ``develop`` branch: ``git checkout develop``.
-- If it's been a while since you created your fork, other people may have updated NOAA's ``develop`` branch. To make sure you're up-to-date, get these changes with ``git pull upstream develop`` and ``git submodule update --recursive --remote``.
-- That command updates the working copy on your computer, but you also need to tell your fork on GitHub about the changes: ``git push origin develop``.
-- Now you're up-to-date and ready to start working on a new feature. ``git checkout -b feature/<my_feature_name>`` will create a new branch (``-b`` flag) off of ``develop`` and switch you to working on that branch.
+1. Switch to the ``develop`` branch
+::
+  git checkout develop
+
+2. Make sure that you pull in changes from the develop branch frequently to simplify the merge process after you submit a PR. Update your local copy (the copy on your computer) of the develop branch
+::
+  git fetch upstream develop
```

Comment:
@bitterbark @jkrasting The command line-only develop update requires adding the GFDL repo as a remote to the git config file.  I will amend this section, and add instructions to use the web interface to update the develop branch.

Reply:
On second thought I think the command line is the way to go:

1.  Devs will need a copy locally anyway
2.  The github base nomenlature and making sure arrows point the right direction, etc. is really confusing to beginners